setup.py

```diff
@@ -23,11 +23,15 @@
 
 from setuptools import find_packages
 
+
+REQUIREMENTS = open('requirements.txt').read().splitlines()
```

Comment:
I'd suggest using more strict parsing here. While base contents of `requirements.txt` is similar to what `install_requires` accepts, file contains also comments (and other elements). You can extract requirements with `pip.req.parse_requirements`: 
https://github.com/pypa/pip/blob/master/src/pip/_internal/req/req_file.py#L60

Reply:
Ok done